### PR TITLE
Webpack optimization

### PR DESCRIPTION
**Pull request by mirpo**
- Updated Webpack
- Improved speed for local development
- Reduced size of dist

### CODE REVIEW

File: src/Connection.js [hunk 1]
```diff
@@ -6,7 +6,16 @@ import { ControlLayer } from 'streamr-client-protocol'
 
 const debug = debugFactory('StreamrClient::Connection')
 
-class Connection extends EventEmitter {
+export default class Connection extends EventEmitter {
+    static get State() {
```
Comment (timoxley) [inline src/Connection.js]: Does this need to be a getter? Means every *access* to `Connection.State` instantiates a new `Object`

File: webpack.config.js [hunk 2]
```diff
@@ -1,17 +1,22 @@
 /* eslint-disable prefer-template */
 /* eslint-disable prefer-destructuring */
 
+process.env.NODE_ENV = process.env.NODE_ENV || 'development' // set a default NODE_ENV
+
 const path = require('path')
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+const webpack = require('webpack')
+const TerserPlugin = require('terser-webpack-plugin')
```
Comment (timoxley) [inline webpack.config.js]: Should this be in package.json?

File: src/Connection.js (hunk 1 above)
Comment (mirpo) [reply src/Connection.js]: You're right, I copied idea from Subscription.js for consistency, but it was wrong idea. Updated both now.

File: webpack.config.js (hunk 2 above)
Comment (mirpo) [reply webpack.config.js]: Yes

**Pull request merged**